### PR TITLE
Release Cabal-3.10.2.1

### PR DESCRIPTION
**Pull request by Kleidukos**
Release Cabal-3.10.2.1

### CODE REVIEW

**Comment by ulysses4ever** [comment]
Why do we need another release shortly after the previous one? Just for that one fix of `c-sources` (https://github.com/haskell/cabal/pull/9285)? This feels a little excessive to me for one fix. Admittedly, I'm not a(n active) user of the said field, so maybe I'm missing how important it is. Nevertheless some justification would be nice to hear.

Will there be a corresponding `cabal-install` release? Or will we get into that slightly awkward state where versions of `cabal-install` and `Cabal` don't match? This is not bad, perhaps, but I remember some aversion to this sort of thing in the past.

**Comment by Kleidukos** [comment]
> Why do we need another release shortly after the previous one? Just for that one fix of c-sources (https://github.com/haskell/cabal/pull/9285)? This feels a little excessive to me for one fix. Admittedly, I'm not a(n active) user of the said field, so maybe I'm missing how important it is. Nevertheless some justification would be nice to hear.

Because it would be too hard of a breaking change to release a cabal-install that has the behaviour used in 3.10.2.0. PR #9285 makes things more ergonomic and prepares for the bigger change that will happen in 3.12.

**Review by ulysses4ever — APPROVED** [review]
Cool with me, thanks!

**Comment by ulysses4ever** [comment]
Mmm, I'd still wait for another approval, just to be on the safe side. There's no rush with it, I don't think.

File: release-notes/Cabal-3.10.2.1.md
Comment (ffaf1) [inline release-notes/Cabal-3.10.2.1.md:4]: Should this be 3.10.2.1?

**Comment by juhp** [comment]
The 3.10 branch is missing #9134 which is a pretty critical pkgconf fix for cabal-install-solver

**Pull request merged**
